### ISSUE TITLE
Midterm Peer Review

### ISSUE THREAD

**Issue by HTGchill** [Yazmine1213C/ORIE5741_Project#9]
The project is about discovery on the knowledge graph structural relations using GNN (Graph Neural Network) on music genres classification problem. The original data is GTZAN (Tzanetakis and Cook(2002)) in this project, collected in 2000-2001. The project can be widely applied in music genres classification.
Things that I like very much:
1.	Application of PCA that visualize entries clearly.
2.	Potential implement on adding more labels as only 2 features seem not so reasonable.
3.	Application of Control Burn to reduce unnecessary features.
Can be improved:
1.	A dataset with later version may be better, 2000 – 2001 seems to be kind of out of date. Or the data’s authority explanation is expected.
2.	The report ought not to be longer than 3 pages.
3.	“Total data is splitted into training and validation by 77%, 33%, respectively” inferring that you are using overlapped training and validation set? Seems kind of strange and it would be better to describe your test set too.
